OpenRA.Mods.Common/UtilityCommands/ExtractLuaDocsCommand.cs

```diff
@@ -77,6 +77,9 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 				foreach (var m in members.OrderBy(m => m.Name))
 				{
 					var desc = Utility.HasAttribute<DescAttribute>(m) ? Utility.GetCustomAttributes<DescAttribute>(m, true).First().Lines.JoinWith("<br />") : "";
+					if (Utility.HasAttribute<ObsoleteAttribute>(m))
+						desc += $"<br />**[DEPRECATED] {Utility.GetCustomAttributes<ObsoleteAttribute>(m, true).First().Message}**";
+
 					Console.WriteLine($"| **{m.LuaDocString()}** | {desc} |");
```

Comment:
```suggestion
					if (Utility.HasAttribute<ObsoleteAttribute>(m))
						Console.WriteLine($"| <s>{m.LuaDocString()}</s> | {desc} |");
					else
						Console.WriteLine($"| **{m.LuaDocString()}** | {desc} |");
```

Use a strike through like most IDEs do.